Library/Homebrew/cmd/doctor.rb

```diff
@@ -92,6 +92,26 @@ def check_for_macgpg2
     end
   end
 
+  # Anaconda installs multiple system & brew dupes, including OpenSSL, Python,
+  # sqlite, libpng, Qt, etc. Regularly breaks compile on Vim, MacVim and others.
+  # Is flagged as part of the *-config script checks below, but people seem
+  # to ignore those as warnings rather than extremely likely breakage.
+  def check_for_anaconda
+    return unless which("anaconda")
+    return unless which("conda-server")
+
+    if which("anaconda") then <<-EOS.undent
```

Comment:
Cool. Do you want to just check Python for now since it's the core point of failure thus far, or?